docs/guides/admin/docs/configuration/elasticsearch.md

```diff
@@ -0,0 +1,35 @@
+Elasticsearch Configuration
+===========================
+
+Elasticsearch powers the external API as well as the administrative user interface of Opencast.
+
+Running Elasticsearch
+---------------------
+
+When running Elasticsearch, it is mandatory to deploy the same major version as the client library used by Opencast has,
+since the client commands will otherwise not match the server. To check the version, take a look at
+[the maven dependency declaration for the elasticsearch bundle in the search module](https://github.com/opencast/opencast/blob/develop/modules/search/pom.xml).
```

Comment:
Long phrase that's also still trying to be informative: my instinct after reading "take a look at" is to immediately click the link, and not read to the end of the sentence before clicking away x)
```suggestion
the maven dependency declaration for the elasticsearch bundle in the [search module](https://github.com/opencast/opencast/blob/develop/modules/search/pom.xml).
```

Reply:
The link is (almost immediately) wrong since the version in develop is likely *not* the version in 9.x.  Or at least it won't be in a few months.